spec/Vehicle/Vehicle.vspec

```diff
@@ -191,6 +191,24 @@ AmbientAirTemperature:
   unit: celsius
   description: Ambient air temperature outside the vehicle.
 
+AmbientHumidity:
```

Comment:
The next signal is named OutsideLightIntensity. Should this signal be OutsideAmbientHumidity?

How about Exterior as a name instead of Outside? Or even making Exterior a branch? As in:

Vehicle.Exterior.AmbientHumidity
Vehicel.Exterior.LightIntensity

Reply:
I understand your point that the names proposed are not that aligned. We already have the signal `AmbientAirTemperature`. I can see a benefit in using the same "prefix" for all 3 signals (temperature, humidity, light intensity), i.e. that all of them start with "Ambient", "Outside" or "Exterior". I have no strong opinion on what to use, or if it shall be a branch or not. One (minor) advantage sticking to "Ambient" is that we avoid the need to rename `AmbientAirTemperature`.

I would have no problems changing `OutsideLightIntensity` to `AmbientLightIntensity`, or to use "Outside" for all, e.g. `OutsideAirTemperature` and `OutsideHumidity`. But `OutsideAmbientHumidity` is a bit redundant, or?

Reply:
I sort of like the idea of an exterior branch to group the signals, even though it would mean breaking the existing temperature signal.

Reply:
Vehicle.Exterior.AmbientHumidity
Vehicle.Exterior.AmbientAirTemperature
Vehicle.Exterior.AmbientLightIntensity

or

Vehicle.Exterior.Humidity
Vehicle.Exterior.AirTemperature
Vehicle.Exterior.LightIntensity